nengo/params.py

```diff
@@ -416,7 +417,7 @@ def validate(self, instance, function_info):
         super(FunctionParam, self).validate(instance, function)
 
 
-class FrozenObject(object):
+class FrozenObject(with_metaclass(DocstringInheritor)):
```

Comment:
Sometimes we just have to have faith in the gods of programming. Surely they have a glorious plan for us!